### ISSUE TITLE
Update HPAT to Julia 0.6 

### ISSUE THREAD

**Issue by rahulkp220** [IntelLabs/HPAT.jl#25]
Hello, First of all, some really great stuff. 
However, with Julia moving to 0.6 and above, do you wish to update the code to match it up with the latest advancements and hence probably fixing the build errors too?

**Comment by ehsantn** [comment]
@rahulkp220 unfortunately, we are not maintaining this repository due to lack of resources. The python version is being developed currently.

**Comment by rahulkp220** [comment]
@ehsantn This is sad 😞 
I saw you guys doing HPAT actively for python which I appreciate, but I think with Julia makes a good candidate for this kind of a library. I hope you agree.

**Comment by ehsantn** [comment]
I agree. Hopefully, the Julia community with pick this up and develop it at some point.
